src/components/Typeahead/Typeahead.scss

```diff
@@ -21,23 +25,24 @@
       left: 0;
 
       li.footer {
-        padding: @gutter-normal;
+        padding: $gutter-normal;
         border-top: solid 1px var(--gray-40);
         margin: 0;
         background-color: var(--pacific-20);
         cursor: default;
-        font-size: @size-vi;
+        font-size: $size-vi;
         font-style: italic;
         list-style-type: none;
         text-align: center;
       }
     }
 
     input {
-      &:extend(.a-text-input);
-      &:focus {
-        &:extend(.a-text-input:focus);
-      }
+      // @extend %a-text-input;
+
+      /* &:focus {
+        @extend .a-text-input:focus;
+      } */
```

Comment:
can we delete these if we dont need them?

Reply:
This has been taken out